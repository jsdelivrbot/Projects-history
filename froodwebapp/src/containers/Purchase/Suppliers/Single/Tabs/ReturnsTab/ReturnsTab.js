import React from 'react';
import { connect } from 'react-redux';
import {
  taxCategoriesGetRequest,
} from 'redux-base/actions';
// import fields from '../../modalFields';
// import styles from '../../Supplier.scss';

const mapStateToProps = state => ({
  needReloadTaxCategories: state.taxes.needReloadTaxCategories,
  loadingPage: state.taxes.loadingPage,
  taxCategories: state.taxes.taxCategories,
  taxCodes: state.taxes.taxCodes,
});

const mapDispatchToProps = {
  taxCategoriesGetRequest,
};

export const ReturnsTab = () => (
  <div>
    <h1>Orders</h1>
  </div>
);

ReturnsTab.propTypes = { };

export default connect(mapStateToProps, mapDispatchToProps)(ReturnsTab);
