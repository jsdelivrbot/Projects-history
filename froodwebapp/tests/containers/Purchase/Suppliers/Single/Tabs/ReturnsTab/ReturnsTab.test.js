import { ReturnsTab } from 'containers/Purchase/Suppliers/Single/Tabs/ReturnsTab/ReturnsTab';
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

const { describe, expect, it } = global;

describe('ReturnsTab', () => {
  const props = {
    loadingPage: false,
    needReloadTaxCategories: false,
    taxCategories: [],
    taxCodes: [],
    taxCategoriesGetRequest: jest.fn()
  };

  it('renders the ReturnsTab component', () => {
    const component = shallow(<ReturnsTab { ...props } />);

    expect(component.length).toEqual(1);
  });

  it('capturing Snapshot of ReturnsTab', () => {
    const component = shallow(<ReturnsTab { ...props } />);
    const view = toJson(component);
    const actual = expect(view);

    actual.toMatchSnapshot();
  });
});
