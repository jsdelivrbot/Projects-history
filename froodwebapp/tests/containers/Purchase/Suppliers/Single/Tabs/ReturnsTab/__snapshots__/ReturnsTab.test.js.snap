// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReturnsTab capturing Snapshot of ReturnsTab 1`] = `
<div>
  <h1>
    Orders
  </h1>
</div>
`;
